zulipterminal/server_url.py

```diff
@@ -1,3 +1,7 @@
+"""
+Constructs and encodes server_url of messages.
```

Comment:
Could be more detailed?

In a few of these, seems like there's no new info added other than what could be assumed from the filename.